sound/mpd/Makefile

```diff
@@ -100,90 +99,110 @@ endef
 
 EXTRA_LDFLAGS += $(if $(ICONV_FULL),-liconv,-Wl,--whole-archive -liconv -Wl,--no-whole-archive) -Wl,-rpath-link=$(STAGING_DIR)/usr/lib/pulseaudio
 
-CONFIGURE_ARGS += \
-	$(call autoconf_bool,CONFIG_IPV6,ipv6) \
-	--disable-debug \
-	--disable-documentation \
-	--disable-test \
-	--disable-aac \
-	--disable-adplug \
-	--disable-ao \
-	--disable-audiofile \
-	--disable-bzip2 \
-	--disable-cdio-paranoia \
-	--disable-fluidsynth \
-	--disable-wildmidi \
-	--disable-gme \
-	--enable-inotify \
-	--disable-icu \
-	--enable-iconv \
-	--disable-iso9660 \
-	--disable-jack \
-	--disable-roar \
-	--disable-libwrap \
-	--disable-lsr \
-	--disable-mad \
-	--disable-mikmod \
-	--disable-modplug \
-	--disable-mpc \
-	--disable-nfs \
-	--disable-openal \
-	--disable-opus \
-	--disable-sidplay \
-	--disable-smbclient \
-	--disable-sndfile \
-	--disable-solaris-output \
-	--disable-sqlite \
-	--disable-systemd-daemon \
-	--disable-lame-encoder \
-	--disable-twolame-encoder \
-	--disable-shine-encoder \
-	--disable-vorbis-encoder \
-	--enable-wave-encoder \
-	--disable-wavpack \
-	--enable-webdav \
-	--disable-zzip \
-	--with-zeroconf=no \
-	--disable-soxr \
-	--enable-curl \
-	--enable-httpd-output \
-	$(call autoconf_bool,CONFIG_AUDIO_SUPPORT,alsa) \
-	--enable-tcp \
-	--disable-sndio \
-	--disable-haiku
+MESON_ARGS += \
+	-Ddocumentation=false \
+	-Dtest=false \
+	-Dsyslog=enabled \
+	-Dinotify=true \
+	-Ddaemon=false \
+	-Dsystemd=disabled \
+	-Dtcp=true \
+	-Dipv6=$(if $(CONFIG_IPV6),enabled,disabled) \
+	-Dlocal_socket=true \
+	-Ddsd=false \
+	-Ddatabase=true \
+	-Dlibmpdclient=enabled \
+	-Dneighbor=false \
+	-Dudisks=disabled \
+	-Dwebdav=enabled \
+	-Dcue=true \
+	-Dcdio_paranoia=disabled \
+	-Dcurl=enabled \
+	-Dnfs=disabled \
+	-Dsmbclient=disabled \
+	-Dqobuz=disabled \
+	-Dtidal=disabled \
+	-Dbzip2=disabled \
+	-Diso9660=disabled \
+	-Dzzip=disabled \
+	-Dchromaprint=disabled \
+	-Dadplug=disabled \
+	-Daudiofile=disabled \
+	-Dfaad=disabled \
+	-Dflac=enabled \
+	-Dfluidsynth=disabled \
+	-Dgme=disabled \
+	-Dmad=disabled \
+	-Dmikmod=disabled \
+	-Dmodplug=disabled \
+	-Dmpcdec=disabled \
+	-Dopus=disabled \
```

Comment:
Interesting that opus is disabled... I wonder if this is from back then libopus was not available in the tree.

Reply:
Or maybe libffmpeg decodes it.

Reply:
Unless you need to encode just let ffmpeg handle everything

Reply:
I too noticed that opus wasn't enabled and figured that's from a time where that package wasn't available.
My idea was to extend this package further after this MR to allow it, this just keeps the exact same feature sets with meson.